core/templates/dev/head/collection_editor/settings_tab/CollectionDetailsEditorDirective.js

```diff
@@ -99,6 +106,21 @@ oppia.directive('collectionDetailsEditor', [function() {
           CollectionUpdateService.setCollectionCategory(
             $scope.collection, $scope.displayedCollectionCategory);
         };
+
+        $scope.updateCollectionLanguageCode = function() {
+          if (!$scope.displayedCollectionLanguage) {
```

Comment:
when will this be triggered? the language selector doesn't let you put custom input.